test/support/factory.ex

```diff
@@ -143,4 +143,422 @@ defmodule ElixirBench.Factory do
       }
     }
   end
+
+  def pull_request_payload() do
```

Comment:
a link to where you got this from would be nice imo - also not sure we should/need to keep all of it. I'd also move it out of this file to not clutter the other factory stuff (it's quite huge) and rather move it into its own little github factory file